back/app/controllers/application_controller.rb

```diff
@@ -59,6 +59,16 @@ def fastjson_params(extra_params = {})
     { current_user: current_user, **extra_params.symbolize_keys }
   end
 
+  def raw_json(json, type: nil)
+    type ||= action_name
```

Comment:
Isn't `action_name` always `nil`? I can't see how it would have any other value?

Reply:
The action name corresponds with the last part of the path of the endpoint. For example, if the endpoint is `GET web_api/v1/stats/ideas_by_status`, then Rails will set the value of `action_name` to `"ideas_by_status"` throughout the controller (unless the route maps it to an action with a different name). We agreed we would use this last part of the path as the type by default. This avoids having to explicitly pass the `type` for each call of `raw_json`.